task/pylint/0.2/tests/run.yaml

```
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: python-test-pipeline
+spec:
+  workspaces:
+    - name: shared-workspace
+  tasks:
+    - name: fetch-repository
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: url
+          value: https://github.com/wumaxd/pylint-pytest-example.git
+        - name: subdirectory
+          value: ""
+        - name: deleteExisting
+          value: "true"
+
+    - name: pylint
+      taskRef:
+        name: pylint
+      runAfter:
+        - fetch-repository
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+      params:
+        - name: args
+          value: ["-r", "y", "src"]
```

Adjusted setup with the path in it
```suggestion
      params:
        - name: args
          value: ["-r", "y"]
        - name: path
          value: "src"
```